src/code/sched.c

```diff
@@ -52,27 +52,37 @@ OSTime sRSPAudioTimeStart;
 OSTime sRSPOtherTimeStart;
 OSTime sRDPTimeStart;
 
+#if OOT_DEBUG
+#define SCHED_PRINTF \
+    if (sLogScheduler)  \
+    PRINTF
+#elif defined(__sgi)
+// See comment about PRINTF in macros.h
+#define SCHED_PRINTF(args) (void)0
+#else
+#define SCHED_PRINTF(format, ...) (void)0
```

Comment:
Maybe should be called `SCHED_DEBUG_PRINTF` like the actor one (and `sLogScheduler` -> `sSchedDebugPrintfEnabled` or something)

Reply:
agree on both